user_guide_src/source/libraries/curlrequest.rst

```diff
@@ -69,9 +69,9 @@ a Response instance to you. This takes the HTTP method, the url and an array of
 Since the response is an instance of ``CodeIgniter\HTTP\Response`` you have all of the normal information
 available to you::
 
-	echo $response->statusCode();
-	echo $response->body();
-	echo $response->header('Content-Type');
+	echo $response->getStatusCode();
+	echo $response->getBody();
+	echo $response->getHeader('Content-Type');
```

Comment:
Currently `getHeader()` is returning a `CodeIgniter\HTTP\Header` object with `var_dump()` but a blank line with `echo`. Perhaps to update it to `getHeaderLine()`.

Reply:
Or you could show **both** examples. They are, after all, just examples of what people can do with it. I'm fine either way, honestly.